packages/core/src/components/typography/typography.tsx

```diff
@@ -79,8 +79,7 @@ export const H6: React.FC<TagProps> = ({ ...props }) => {
 };
 
 export const P: React.FC<TagProps> = ({ testId, ...props }) => {
-	return <Text tag="p" data-testid={testId}
-	{...props} />;
+	return <Text tag="p" data-testid={testId} {...props} />;
```

Comment:
if you've changed `testid` to lowercase, you'll need to change it here too instead of the camelCase format.

Reply:
Resolved in latest commit